docs/steps-elements.md

```diff
@@ -201,12 +201,55 @@ Allows to check if `:elementName` has a value that matches the `:matcher`.
 
 ---
 
+## `there is element ":elementName" containing ":matcher" text`
+
+Allows to check if `:elementName` contains a text that matches the `:matcher`.
+
+---
+
+## `there is element ":elementName" matching ":matcher" matcher`
+
+Allows to check if `:elementName` matches the given type of `:matcher`. For example:
```

Comment:
Same here, that's good we have an example but the description should explain that:
- this is an alias of another step
- you can use all matchers e.g. isVisible without prefix `f:`